src/main/java/com/synopsys/integration/alert/common/descriptor/Descriptor.java

```diff
@@ -110,12 +121,25 @@ public boolean hasUIConfigForType(final ConfigContextEnum actionApiType) {
     }
 
     public void validateConfig(final ConfigContextEnum actionApiType, final FieldAccessor fieldAccessor, final Map<String, String> fieldErrors) {
-        getRestApi(actionApiType).validateConfig(fieldAccessor, fieldErrors);
+        final Optional<DescriptorActionApi> actionApi = getActionApi(actionApiType);
```

Comment:
This, along with several other methods here could be replaced by fluent Optional methods.
e.g.
```
getActionApi(actionApiType).ifPresent(api -> api.validateConfig(fieldAccessor, fieldErrors));
```

Reply:
Couldn't change them all because of exceptions